tests/unit/include/bout/test_petsc_indexer.cxx

```diff
@@ -126,16 +127,16 @@ TYPED_TEST(IndexerTest, TestConvertIndex) {
 
 TYPED_TEST(IndexerTest, TestIsLocal) {
   BOUT_FOR(i, this->globalSquareIndexer.getRegionAll()) {
-    EXPECT_TRUE(this->globalSquareIndexer.isLocal(i));
+    ASSERT_TRUE(this->globalSquareIndexer.isLocal(i));
   }
   BOUT_FOR(i, this->globalStarIndexer.getRegionAll()) {
-    EXPECT_TRUE(this->globalStarIndexer.isLocal(i));
+    ASSERT_TRUE(this->globalStarIndexer.isLocal(i));
   }
   BOUT_FOR(i, this->globalDefaultIndexer.getRegionAll()) {
-    EXPECT_TRUE(this->globalDefaultIndexer.isLocal(i));
+    ASSERT_TRUE(this->globalDefaultIndexer.isLocal(i));
   }
   BOUT_FOR(i, this->localIndexer.getRegionAll()) {
-    EXPECT_TRUE(this->localIndexer.isLocal(i));
+    ASSERT_TRUE(this->localIndexer.isLocal(i));
```

Comment:
I get `error: invalid exit from OpenMP structured block` on all these. `BOUT_FOR` -> `BOUT_FOR_SERIAL` fixes it

Reply:
This was an error I got on PR #1873 which I had fixed there. Somehow that fix got reverted here. Using EXPECT_TRUE works fine and I have commited that on my machine.